### DISCUSSION TITLE
[应用商店建议]程序管理

### DISCUSSION THREAD

**Discussion by 157221810** [linuxdeepin/developer-center#3016] [Features Request & Ideas | 特性请求 & 头脑风暴]
现状：本机安装太多的应用，当想删除某应用时，在应用商店的`管理`里，难以查找。因为在这里的搜索出来的结果，只有`打开`操作选项，哪怕对搜索到的结果点进去，也无法找到‘缷载’操作。
即：在‘管理’栏中的搜索逻辑是混乱的
改进建议：
1、搜索出的结果，如果已经安装了，则在相应app的‘打开’操作旁边，给出‘卸载’操作；也可以在app明细界面增加'卸载'操作；
2、在‘管理’栏下，搜索出的结果，是与‘管理’栏目下的apps的交集，亦即在我已安装的程序中搜索，而非在整个商店中搜索应用，因为那些结果与我安装的应用无关；
以上两条取一即可，亦或有更好的改进方式。

**Comment by meiyixiang** [comment]
![image](https://user-images.githubusercontent.com/75475994/164204569-8824cb0c-849c-4d34-a0c7-a2ff4ddbbd1e.png)
I don't know if I have a problem with that, but I think you feel you should be able to provide easier management of installed software in the store.



As shown in the figure 1, the installed software is in the management menu. Corresponding to each installed software, 3 can provide the uninstall function
